Day-I-Social-Card/social-card/src/components/CardComponents/CardContent.js

```diff
@@ -1,2 +1,14 @@
 import React from 'react';
 import './Card.css';
+
```

Comment:
After you get a little more practice with `state` you should come back and see if you can make this component reusable. Keep simplicity in mind when you build this out. The more complicated you go the harder it will be to debug.